modules/rosa-upgrading-automatic-ocm.adoc

```diff
@@ -5,16 +5,18 @@
 
 :_content-type: PROCEDURE
 [id="rosa-scheduling-upgrade_{context}"]
-= Scheduling automatic upgrades
+= Scheduling recurring upgrades for your cluster
 
-You can schedule automatic upgrades for a {product-title} cluster through {cluster-manager-first} console.
+You can schedule recurring, automatic upgrades for your {product-title} cluster on a weekly basis through the {cluster-manager} console.
```

Comment:
Maybe I am being too particular. But this sentence can read that on a weekly basis, you can schedule upgrades. Will the sentence still make sense without _on a weekly basis_?

Reply:
It can. Made the change.